Explorer/ProjectSettings/QualitySettings.asset

```diff
@@ -126,7 +128,7 @@ QualitySettings:
     globalTextureMipmapLimit: 0
     textureMipmapLimitSettings: []
     anisotropicTextures: 2
-    antiAliasing: 8
+    antiAliasing: 4
```

Comment:
Is this antialiasing change correct? 

Why did this QualitySettings.asset file change at all?